CMakeLists.txt

```diff
@@ -314,6 +318,25 @@ option(NO_DX "Disable DirectX support")
 option(NO_TESTS "Disable all tests")
 option(NO_GLTESTS "Disable GL tests")
 
+option(MSVC_STATIC_CRT_RELEASE "Statically link MSVC CRT in Release" OFF)
```

Comment:
Not sure what option name you'd prefer. This naming convention goes against the existing `NO_` options. Could invert them if preferable. `NO_DYNAMIC_MSVC_CRT` etc.